native/dpotf2.go

```diff
@@ -0,0 +1,66 @@
+package native
+
+import (
+	"math"
+
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+)
+
+// Dpotrf computes the cholesky decomposition of the symmetric positive definite
+// matrix a. If ul == blas.Upper, then a is stored as an upper-triangular matrix,
+// and a = U^T U is stored in place into a. If ul == blas.Lower, then a = L L^T
+// is computed and stored in-place into a. This is the unblocked version of the
+// algorithm. Dpotrf returns 0 if the decomposition is successful, otherwise
```

Comment:
We make essentially the same degree of changes in cgo/blas with the rot procedures. I don't see an issue with that.

This is pretty much equivalent to how the standard library wraps ERRNO.